src/index.ts

```diff
@@ -188,7 +211,13 @@ export default class IncludeFragmentElement extends HTMLElement {
       if (!isWildcard(this.accept) && (!ct || !ct.includes(this.accept ? this.accept : 'text/html'))) {
         throw new Error(`Failed to load resource: expected ${this.accept || 'text/html'} but was ${ct}`)
       }
-      const data = await response.text()
+
+      const responseText: string = await response.text()
+      let data: string | CSPTrustedHTMLToStringable = responseText
+      if (cspTrustedTypesPolicyPromise) {
+        const cspTrustedTypesPolicy = await cspTrustedTypesPolicyPromise
+        data = cspTrustedTypesPolicy.createHTML(responseText, response)
```

Comment:
If the policy throws an error, this passes it on verbatim. Should we catch the error and wrap it in something like `CSP trusted types policy failed with error: ${oldError}`?

Reply:
I think passing the error verbatim would be acceptable.